automation_tools/__init__.py

```diff
@@ -338,6 +338,10 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
         proxy = 'capsule'
     else:
         proxy = 'foreman-proxy'
+    if os.environ.get('SATELLITE_VERSION') in ('6.0', '6.1', '6.2', '6.3'):
```

Comment:
Are you sure that the command changed for all versions of Satellite? I can attest that Satellite 6.2.7 shows several entries for `--foreman-proxy'